.changeset/search-bobcats-love.md

```diff
@@ -0,0 +1,98 @@
+---
+'@backstage/plugin-search-react': minor
```

Comment:
I have a feeling a common use-case for the limit will be setting the default value on the `initialValue` of `<SearchContextProvider>`, rather than exposing UI-based control over it.

It'd be great to highlight it as an option, at least.

Reply:
I updated the changeset file and also the pull request description, does it look good to you?